src/main.cpp

```diff
@@ -3996,14 +3996,18 @@ std::string GetWarnings(const std::string& strFor)
     if (fLargeWorkForkFound)
     {
         nPriority = 2000;
-        strStatusBar = strRPC = _("Warning: The network does not appear to fully agree! Some miners appear to be experiencing issues.");
+        strStatusBar = strRPC = "Warning: The network does not appear to fully agree! Some miners appear to be experiencing issues.";
     }
     else if (fLargeWorkInvalidChainFound)
     {
         nPriority = 2000;
-        strStatusBar = strRPC = _("Warning: We do not appear to fully agree with our peers! You may need to upgrade, or other nodes may need to upgrade.");
+        strStatusBar = strRPC = "Warning: We do not appear to fully agree with our peers! You may need to upgrade, or other nodes may need to upgrade.";
     }
 
+    // try to i18n the string
+    if (translate)
+        strStatusBar = _(strStatusBar.c_str());
```

Comment:
We might want to keep the info string on the overview page i18n'ed while avoiding translation on RPC level (check the screen):

<img width="938" alt="bildschirmfoto 2015-11-30 um 14 45 39" src="https://cloud.githubusercontent.com/assets/178464/11473588/8e036990-9773-11e5-8180-be395c77789c.png">